prbt_default_application/scripts/myFirstApplication.py

```diff
@@ -23,8 +23,8 @@ def start_program():
     r.move(Ptp(goal=start_pos, vel_scale=__ROBOT_VELOCITY__))
 
     rospy.loginfo("Start loop") # log
-    while(True):
-        # do infinite loop
+    for _ in range(5):
```

Comment:
Why? An application should run as long as the application runs. In C++ the usual use is `while(ros::ok()), in python it's a pure `while True:`.
Inside the loop, the user would e.g. wait, unitl parts from a feeder are available.